### PR TITLE
Splash/Home page

### PR DESCRIPTION
**Pull request by meagancheese**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -16,7 +16,18 @@
...
     <div id="content">
-
+      <!--TODO: Add a navbar-->
+      <p id="welcome">Welcome to InterviewMe!</p>
+      <p>InterviewMe is a platform for conducting mock technical interviews where interviewers and interviewees alike can 
+        practice their skills. We aim to connect users with live video chats in order to encapsulate the true feeling
+        and pace of a technical interview.
+      </p>
+      <br>
+      <p>If you would like to logout, please click here.</p>
```
Comment (apluscs) [inline src/main/webapp/index.html]: l think we should incorporate login/logout as one portion of the navbar. Delete this?

File: target/maven-archiver/pom.properties [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#Generated by Maven
+<<<<<<< HEAD
+#Tue Jun 23 15:32:43 UTC 2020
+=======
+#Tue Jun 23 15:42:06 UTC 2020
+>>>>>>> 1b64b618d0eb3b78635f628ae690150fb59b9173
```
Comment (apluscs) [inline target/maven-archiver/pom.properties]: Seems like a conflict to address.

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
-    width: 650px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#welcome {
+  font-size: 18;
+  text-align: center;
+  margin-bottom: 10px;
```
Comment (apluscs) [inline src/main/webapp/style.css]: Not a fan of specifying font sizes myself. l prefer using h1, h2, etc. to denote different font sizes. lt's more elegant that way and easier to track font sizes. l think we should try to Bootstrap as much as we can in general.
https://getbootstrap.com/docs/4.0/content/typography/

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -16,7 +16,18 @@
   </head>
   <body>
     <div id="content">
-
+      <!--TODO: Add a navbar-->
+      <p id="welcome">Welcome to InterviewMe!</p>
```
Comment (apluscs) [inline src/main/webapp/index.html]: change id to something more specific? like "welcome-heading"?

**Review by apluscs — CHANGES_REQUESTED** [review]
l think it'd be great if we could get into the habit of attaching screenshots, especially when large changes are made to a page. Attach one please?

File: src/main/webapp/index.html (hunk 1 above)
Comment (meagancheese) [reply src/main/webapp/index.html]: Done

File: target/maven-archiver/pom.properties (hunk 2 above)
Comment (meagancheese) [reply target/maven-archiver/pom.properties]: Done

File: src/main/webapp/index.html (hunk 4 above)
Comment (meagancheese) [reply src/main/webapp/index.html]: Done

File: src/main/webapp/style.css (hunk 3 above)
Comment (meagancheese) [reply src/main/webapp/style.css]: Done

**Comment by meagancheese** [comment]
Good call, here you go: 
Screenshot 2020-06-23 at 5 21 18 PM

**Review by apluscs — CHANGES_REQUESTED** [review]
For the Home Page, l think it looks better if the text is centered. Also the title is too big in comparison to the subtitle. Make the title smaller and the subtitle a larger? 
l usually model pages that need to be aesthetic after something professional like the ones here: https://line25.com/articles/showcase-of-web-designs-with-modern-splash-pages
Can't go wrong there!

**Review by apluscs — APPROVED** [review]
Love the look! LGTM.

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -16,7 +16,31 @@
   </head>
   <body>
     <div id="content">
-
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <span class="navbar-brand mb-0 h1">InterviewMe</span>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (jalexanderqed) [inline src/main/webapp/index.html]: You should split this <button> tag, possibly twice, so that it is not overly long.

**Review by jalexanderqed — APPROVED** [review]
One change requested, but otherwise LGTM

File: src/main/webapp/index.html (hunk 5 above)
Comment (meagancheese) [reply src/main/webapp/index.html]: Done

**Pull request merged**